test/acceptance/tests.py

```diff
@@ -1201,6 +1263,30 @@ def test_feedback_only(self):
         self.staff_area_page.verify_learner_final_score("Final grade: 1 out of 1")
 
 
+@ddt.ddt
+class MultipleOpenAssessmentTest(OpenAssessmentTest, MultipeOpenAssessmentMixin):
+    """
+    Test the multiple peer-assessment flow.
+    """
+
+    def setUp(self):
+        super(MultipleOpenAssessmentTest, self).setUp('multiple_ora')
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
```

Comment:
Echoing @cahrens' comment here - how are the different staff area pages being differentiated? I'm going to run this locally and see what's happening